src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -84,6 +84,15 @@
     <PackageReference Include="System.Collections.Immutable" Version="8.0.*-*" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' Or ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' &lt; '8.0')" />
   </ItemGroup>
 
+  <!--
+    Workaround for https://github.com/dotnet/performance/issues/3164
+  -->
+  <ItemGroup Condition="$([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '8.0'))">
```

Comment:
This fails with `error MSB4184: The expression "[MSBuild]::VersionGreaterThanOrEquals('', 8.0)" cannot be evaluated. Version string was not in a correct format.`. Better to use `$(_TargetFrameworkVersionWithoutV)` (used couple times above).

Reply:
thanks!